README.md

```diff
@@ -1 +1,116 @@
 # digital-media-object-processor
+The digital media processor objects can receive digital media object from two different sources:
```

Comment:
Would change to: The digital media processor can receive digital media objects from two different sources:

Reply:
:+1: